blueprints/app/files/app/index.html

```diff
@@ -15,6 +15,10 @@
     {{content-for "head-footer"}}
   </head>
   <body>
+    <noscript>
+      JavaScript must be enabled to run this application.
```

Comment:
I am unsure about this. It might just render on the screen, but everything else should work. If there was a way to add a check if fastboot is enabled, I could add that.